moma_mission/src/moma_mission/missions/piloting/manipulation_sim.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import rospy
+from nav_msgs.msg import Path
+from moma_mission.core import StateRosControl
+
+from moma_mission.utils.transforms import *
+from moma_mission.utils.robot import Robot
+
+class ValveManipulationState(StateRosControl):
+    def __init__(self, ns):
+        StateRosControl.__init__(self, ns=ns)
+
+        self.world_frame = self.get_scoped_param("world_frame", "world")
+        self.reference_frame = self.get_scoped_param("reference_frame", "tracking_camera_odom")
+        self.grasp_frame = self.get_scoped_param("grasp_frame", "grasp_point")
+        self.turning_angle = np.deg2rad(
+            self.get_scoped_param("turning_angle_deg", 45.0))
+        self.turning_speed = np.deg2rad(
+            self.get_scoped_param("turning_speed_deg", 5.0))
+
+        self.valve_description_name = self.get_scoped_param(
+            "valve_description_name", "valve_description")
+        self.pinocchio_robot = Robot(self.valve_description_name)
+
+        path_topic_name = self.get_scoped_param("path_topic_name", "/desired_path")
+        self.path_publisher = rospy.Publisher(path_topic_name, Path, queue_size=1, latch=True)
+
+        self.R_V_EE = R.from_euler('xyz', [0.0, 180.0, -90.0], degrees=True).as_dcm()
```

Comment:
We should do something for this hard coded params. It is difficult to get rid of them but are quite dangerous. I would at least put them into a `constant.py` or similar to avoid duplication and have a single source of truth. 

The alternative is that we have them as params but I dont like this solution as they need to propagate through ros nodes using the param server ... without considering all the issues related to default params.

Reply:
I don't think it makes sense to abstract this parameter to a global constant, as it is very specific to the URDF ("valve_description") we are using. I think making it a rosparam with default serves the purpose here. If we use a different URDF, we are likely to change the "grasp_frame" parameter anyway, and then this rotation parameter can be changed in the same place.